CMakeLists.txt

```diff
@@ -1,8 +1,13 @@
-set(CMAKE_CXX_STANDARD 11)
 cmake_minimum_required(VERSION 3.1.2)
 cmake_policy(SET CMP0015 NEW) # Link Directory Pathing
 set(CMAKE_DEBUG_POSTFIX _d)
 
+if (APPLE)
```

Comment:
Do the windows versions require higher versions of visual studio for this to work for them?

Reply:
The compiler changes were made for MacOS only. It uses the same versions of boost as used previously for windows and linux builds requiring cxx11 (1.60.0 for xms builds and 1.66.0 for other windows and linux builds). Boost 1.66 isn't available from conan-center for MacOS so it uses an aquaveo built boost 1.74.0 using apple-clang 11.0 with cxx17 turned on.